### ISSUE TITLE
npm run update-webdriver FAILS

### ISSUE THREAD

**Issue by nikolaz111** [angular/angular-seed#245]
Hello I'm trting the end to end capabilities of the seed project, but when i run `npm run update-webdriver` the command fails.

```

$ npm run update-webdriver

> angular-seed@0.0.0 preupdate-webdriver /home/nicolas/miido/miido-web
> npm install


> angular-seed@0.0.0 postinstall /home/nicolas/miido/miido-web
> bower install


> angular-seed@0.0.0 update-webdriver /home/nicolas/miido/miido-web
> webdriver-manager update

Updating selenium standalone
downloading https://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar...
Updating chromedriver
downloading https://chromedriver.storage.googleapis.com/2.12/chromedriver_linux64.zip...
Error: Got error Error: 140279323998016:error:0607907F:digital envelope routines:EVP_PKEY_get1_RSA:expecting an rsa key:../deps/openssl/openssl/crypto/evp/p_lib.c:288:
 from https://chromedriver.storage.googleapis.com/2.12/chromedriver_linux64.zip
Error: Got error Error: 140279323998016:error:0607907F:digital envelope routines:EVP_PKEY_get1_RSA:expecting an rsa key:../deps/openssl/openssl/crypto/evp/p_lib.c:288:
 from https://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar

```

I'm running it on a Ubuntu 13.10 machine

What am I missing?

**Comment by michikono** [comment]
The documentation on http://angular.github.io/protractor/#/ shows that you need to run:

`npm install -g protractor`

**Comment by pritchardjonathan** [comment]
updating to that latest version of node did the trick for me

**Comment by JMuhire** [comment]
The command works for me

**Comment by gkalpak** [comment]
Fixed in #341.
